src/app/pages/HeroletesMarketplace/HeroleteItem/HeroleteItem.tsx

```diff
@@ -44,7 +44,7 @@ export const HeroleteItem = ({
             Collection: {collection.split(' -').shift()}
           </Typography>
           <Typography component="span" variant="subtitle2" className={styles.heroleteItem__infoTier}>
-            <RibbonIcon /> {tier}
+            <RibbonIcon fill="#F58550" /> {tier}
```

Comment:
could you use a color from contants file?

Reply:
Ah right! I forgot we have colours over there